CONTRIBUTING.md

```diff
@@ -68,3 +78,20 @@ To release any project in this repository:
 5. Tag the above commit with as `<project>/v<version>` with `git tag $(changie latest -j <project>) <commit-sha>`.
     - If the operator is being released, also tag the same commit as `v<version>`.
 6. Push the tag(s).
+
+## Nightly build
+
+### Schedule builds
+
+Buildkite has configured periodic jobs that build and pushes operator container with operator
+helm chart to https://hub.docker.com/r/redpandadata/redpanda-operator-nightly container repository.
+The following branches are currently build periodically:
+* main
+* release/v2.3.x
+
+### Manual nightly build
+
+In Buildkite anyone with the access to redpanda-operator project can trigger build (upper right corner 
```

Comment:
If you have the screenshots you shared lying around, I think they'd be very helpful to include. You can drop them into `.github/images`.